### PR TITLE
feat(STN- 254):  polish

### PR DESCRIPTION
**Pull request by silverli**
# READY FOR REVIEW

## Summary
Paired with a designer to polish our work: 

- [ ] Main Navigation: check link font sizes and responsive behavior
- [ ] Spacing (margin) in logo lockup in header: view with all school options **check again, test with a longer title 
- [ ] Check dropdown shadow issue on secondary navigation
- [ ] Look at width of search bar at 940px – it should be close to the width of the lockup
- [ ] Mobile margins around page content needs to be decreased: check outer margin on mobile
- [ ] Plan for responsive heading text sizes in both vertical and horizontal cards when resizing for mobile. This may become another card because we need to account for different patterns and the different layout options. “
- [ ] Explore different heading font sizes at vertical 3-column vs 2-column”

TODO browser testing.

## Steps to Test
1. Run `npm run test` and ensure tests pass.
2. Check through the list above and make sure all those details have been fixed and look better!

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss [hunk 1]
```diff
@@ -235,9 +235,36 @@
     }
 
     h2 {
-      @include hb-heading-3;
+      @include hb-heading-5;
       margin: 0;
       overflow-wrap: break-word;
+
+      // If this h2 is in a grid,
+      // keep it smaller longer.
```
Comment (kaseybon) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss:255]: Should we also do this with the Vertical Linked Card Title?

File: docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss [hunk 2]
```diff
@@ -98,6 +98,8 @@
       }
 
       @include grid-media-min('lg') {
+        box-shadow: 0 2px 16px 0 rgba(0, 0, 0, 0.1) !important;
```
Comment (kaseybon) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss:102]: CodeClimate is going to catch this, we need to wrap it in an ignore block.

File: docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss [hunk 3]
```diff
@@ -5,14 +5,31 @@
 
   .su-lockup {
     z-index: $hb-z-index-lockup;
+
```
Comment (kaseybon) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss:8]: Perhaps my lockup isn't setup correctly but I'm getting some funky spacing between the text. Is this expected?
Screen Shot 2020-03-23 at 7 29 25 AM

File: docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss (hunk 1 above)
Comment (silverli) [reply docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss:255]: great call!

**Review by kaseybon — APPROVED** [review]

**Pull request merged**